sdk/template/azure-template/CHANGELOG.md

```diff
@@ -1,15 +1,7 @@
 # Release History
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0-beta.8 (Unreleased)
```

Comment:
Wait for https://github.com/Azure/azure-sdk-for-cpp/pull/762 to go in, before making this version update change on top.

Reply:
762 isn't correct.

Reply:
It is now.